src/components/card/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+interface Props {
+  children?: React.ReactNode
+}
+
+function Card({ children }: Props) {
+  return (
+    <div
+      className="card"
+      style={{
```

Comment:
Are we doing everything inline while we dont make a decision on styling?

Reply:
I am not sure I have just used inline since it would be easy to convert later.